scss/atoms/typography/_typography.scss

```diff
@@ -54,7 +54,7 @@ h6 {
 	font-size: em( 16px );
 	font-weight: 600;
 	line-height: 1;
-	text-transform: uppercase;
+	text-transform: uppercase; // FIXME Is this opinionated?
 }
```

Comment:
@sfrisk I kinda don't link this. Isn't that opionanted? More people would use h6 with normal capitalization than uppercase.